PF/PF_Session_4/N_Queen.java

```diff
@@ -0,0 +1,96 @@
+import java.util.*;
+public class N_Queen
+{
+	static int[] check;					
+	int resultArray[][];				
+	
+	public N_Queen(int numOfQueens) 
+	{
+        check = new int[numOfQueens];		// Array definition with size as noOFQueens
+    }
+	public static void main(String[] args) 
+	{
+		// TODO Auto-generated method stub
+		System.out.println("Enter queens");
```

Comment:
Assignment expects a function " nQueen(board , 0, 4)". which will return a boolean. Please follow the requirement.